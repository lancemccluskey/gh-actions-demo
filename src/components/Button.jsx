/** @jsx jsx */
import { jsx } from 'theme-ui';

const Button = ({ handleNextClick, handlePrevClick }) => {
  return (
    <div sx={{ display: 'flex', justifyContent: 'space-around' }}>
      <button sx={{ backgroundColor: 'highlight' }} onClick={handlePrevClick}>
        Like
      </button>
      <button sx={{ backgroundColor: 'highlight' }} onClick={handleNextClick}>
        Next
      </button>
    </div>
  );
};

export default Button;
