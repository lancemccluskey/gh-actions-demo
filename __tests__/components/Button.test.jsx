import { render, screen } from '@testing-library/react';
import Button from '../../src/components/Button';

describe('<Button />', () => {
  it('renders', () => {
    const { container } = render(<Button />);

    expect(container).toMatchSnapshot();
  });

  it('should have Next and Like', () => {
    render(<Button />);

    expect(screen.getByText('Next')).toBeInTheDocument();
    expect(screen.getByText('Like')).toBeInTheDocument();
  });
});
