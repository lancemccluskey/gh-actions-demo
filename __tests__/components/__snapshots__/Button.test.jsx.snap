// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders 1`] = `
<div>
  <div
    class="css-13uxruf-Button"
  >
    <button
      class="css-uw5q0g-Button"
    >
      Like
    </button>
    <button
      class="css-uw5q0g-Button"
    >
      Next
    </button>
  </div>
</div>
`;
